config/rbac/cluster_monitoring_view_prometheus_k8s_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
```

Comment:
This is not the right file to add custom RBACs. These will be removed when we update generated content.
Add this CRB to [ocs-operator/rbac](https://github.com/red-hat-storage/ocs-operator/tree/6d2b355ea4cc183d493379f88d4f8b5fde3721f2/rbac) directory.

Then run `make update-generated` and `make gen-latest-csv` to update the operator bundle.

Reply:
Done